src/Symfony/Component/DomCrawler/Test/Constraint/CrawlerSelectorAttributeValueSame.php

```diff
@@ -47,7 +48,13 @@ protected function matches($crawler): bool
             return false;
         }
 
-        return $this->expectedText === trim($crawler->getNode(0)->getAttribute($this->attribute));
+        if ($crawler instanceof PantherCrawler) {
```

Comment:
should we add a method in the Crawler to cover that ?

Reply:
We could indeed

Reply:
Not a big fan of having Panther related code here.

Reply:
So can I apply @stof suggestion? It removes the need for the Panther specific code, and allows BrowserKit and Panther to converge a bit more.

Reply:
@stof suggestion makes sense to me.

Reply:
Actually, a similar method already exists (and is implemented by both Panther and BrowserKit): `attr()`. I updated the constraint to use it.